frontend/src/component/operator-wizard/element-component/field-factory/DropdownField.js

```diff
@@ -32,7 +32,9 @@ const mapDispatchToProps = dispatch => {
 
 const stepName = "Components",
   itemName = "Smart Router",
-  selectedEnvPrefix = "rhdm";
+  rhdmEnvPrefix = "rhdm",
+  rhpamEnvPrefix = "rhpam",
+  fieldName = "Environment";
```

Comment:
Rename it to something more evident. When used, `fieldName` does not help the developer to understand the purpose of this constant. Try just `ENV_FIELD_NAME`;